src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
```

Comment:
The class definition should not start with an empty line. Please remove the redundant empty line at the beginning of the class.

Reply:
++